src/routes/(all)/order/+page.svelte

```diff
@@ -44,47 +50,134 @@
       status = "Замовлення відправлено"
```

Comment:
In design you should get modal with message that success. 
**Figma -> Ecommerce -> Order Success**
![image](https://github.com/flurium/spentoday-shop/assets/103932583/e6e4aa73-f2f7-48b5-a101-53d0ae6c001f)